test/data/build-c/solution.csolution.yml

```diff
@@ -20,8 +20,8 @@ solution:
           - -lm
           - -lc
 
-    # - type: IAR
-    #   compiler: IAR
+    - type: IAR
```

Comment:
Typically, we avoid specifying the compilers as build-types.
We should rather use `.cdefault` or `misc` on solution level and `--tolchain` on command line.

Reply:
Yes, these are just the old test examples to distinguish and have different contexts for different compilers